api/class-wc-calypso-bridge-product-reviews-controller.php

```diff
@@ -0,0 +1,385 @@
+			$args['post_id'] = $request['product'];
+		}
+
+		if ( ! empty( $request['search']) ) {
+			$args['search'] = $request['search'];
+		}
+
+		$reviews_query = new WP_Comment_Query;
+		$reviews       = $reviews_query->query( $args );
+
+		$data = array();
+		foreach ( $reviews as $review_data ) {
+			$review = $this->prepare_item_for_response( $review_data, $request );
+			$review = $this->prepare_response_for_collection( $review );
+			$data[] = $review;
+		}
+
+		$response = rest_ensure_response( $data );
+
+		$response->header( 'X-WP-Total', (int) $reviews_query->found_comments );
+		$response->header( 'X-WP-TotalPages', (int) $reviews_query->max_num_pages );
+
+		return $response;
+	}
+
+	/**
+	 * Prepare links for the request.
+	 *
+	 * @param WP_Comment $review Product review object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return array Links for the given product review.
+	 */
+	protected function prepare_links( $review, $request ) {
+		$links = array(
+			'self' => array(
+				'href' => rest_url( sprintf( '/%s/%s', $this->namespace, 'products/' . $review->comment_post_ID . '/reviews/' . $review->comment_ID ) ),
+			),
+			'collection' => array(
+				'href' => rest_url( sprintf( '/%s/%s', $this->namespace, 'products/' . $review->comment_post_ID . '/reviews' ) ),
+			),
+			'up' => array(
+				'href' => rest_url( sprintf( '/%s/%s', $this->namespace, 'products/' . $review->comment_post_ID ) ),
+			),
+		);
+
+		return $links;
+	}
+
+	/**
+	 * Prepare a single product review output for response.
+	 *
+	 * v3 adds 'product_id' and 'approved'.
+	 *
+	 * @param WP_Comment $review Product review object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response $response Response data.
+	 */
+	public function prepare_item_for_response( $review, $request ) {
+		$product = new WC_Product( $review->comment_post_ID );
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+
+		$images        = wp_get_attachment_image_src( $product->get_image_id(), 'full' );
+		$product_image = is_array( $images ) ? current( $images ) : '';
+
+		switch( $review->comment_approved ) {
+			case '1':
+				$status = 'approved';
+				break;
+			case 'spam':
+				$status = 'spam';
+				break;
+			case 'trash':
+				$status = 'trash';
+				break;
+			case '0':
+			default:
+				$status = 'pending';
+				break;
+		}
+
+		$content = 'view' === $context ? wpautop( $review->comment_content ) : $review->comment_content;
+
+		$data = array(
+			'id'               => (int) $review->comment_ID,
+			'date_created'     => wc_rest_prepare_date_response( $review->comment_date ),
+			'date_created_gmt' => wc_rest_prepare_date_response( $review->comment_date_gmt ),
+			'review'           => $content,
+			'rating'           => (int) get_comment_meta( $review->comment_ID, 'rating', true ),
+			'name'             => $review->comment_author,
+			'email'            => $review->comment_author_email,
```

Comment:
Why is email needed? Would be great if we could avoid sharing this personal data.